addons/mrp/models/mrp_workorder.py

```diff
@@ -695,7 +695,7 @@ def _get_duration_expected(self, alternative_workcenter=False, ratio=1):
             if duration_expected_working < 0:
                 duration_expected_working = 0
             return alternative_workcenter.time_start + alternative_workcenter.time_stop + cycle_number * duration_expected_working * 100.0 / alternative_workcenter.time_efficiency
-        time_cycle = self.operation_id and self.operation_id.time_cycle or 60.0
+        time_cycle = self.operation_id.time_cycle
```

Comment:
time_cycle = self.operation_id.time_cycle if self.operation_id else 60

Reply:
Would have the same than X and Y or Z but if if Y is false then Z is used (that's the bug in our case)

Reply:
The case where self.operation_id is `False` or `None` is already covered above, and we return earlier if that's the case https://github.com/odoo/odoo/blob/d06f2583c515c69ac8ad7f6a403e459adc43e410/addons/mrp/models/mrp_workorder.py#L685-L689 so I think that it's unnecessary to check again here, but I might be wrong 😄